src/shipit_static_analysis/shipit_static_analysis/report/mail.py

```diff
@@ -11,10 +11,21 @@
 logger = log.get_logger(__name__)
 
 
-EMAIL_HEADER = '''{nb_publishable} Publishable issues on Mozreview
+EMAIL_STATS_LINE = '|{source}|{total}|{publishable}|'
+
+EMAIL_HEADER = '''
+# Found {total} issues
+
+| Source | Issues | Publishable |
+|--------|--------|-------------|
+{stats_table}
```

Comment:
Nit: I find the generated table pretty hard to read:

![stats-table-problem](https://user-images.githubusercontent.com/599268/34828956-7ec74938-f6df-11e7-9451-49461fd98631.png)

Maybe justify everything toward the left, to align titles and content? Or you could also make this a simple list like so:

- MozLintIssue: 1 publishable (270 total)
- CangTitdyIssue: 5 publishable (600 total)

Reply:
I cannot control the style of the mail (sent by Taskcluster notify service).
So i'll change to display the list as `X publishable (Y total)`